### PR TITLE
docs: add helm-installation 

### PR DESCRIPTION
**Pull request by Joyinqin**
Signed-off-by: Joyinqin <joyinqin@pku.edu.cn>
This pr is translated from: https://github.com/chaos-mesh/website-zh/pull/15

docs: add production-installation-using-helm.md

### CODE REVIEW

File: docs/production-installation-using-helm.md [hunk 1]
```diff
@@ -1,6 +1,208 @@
...
+helm version
+```
+
+The expected output is as follows:
+
+```bash
+version.BuildInfo{Version:"v3.5.4", GitCommit:"1b5edb69df3d3a08df77c9902dc17af864ff05d1", GitTreeState:"dirty", GoVersion: "go1.16.3"}
+```
+
+If your actual outputs are similar to the expected outputs with `Version`, `GitCommit`, `GitTreeState` and `GoVersion`, it means that Helm is installed successfully.
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:27]:
```suggestion
If your actual output is similar to the expected output with `Version`, `GitCommit`, `GitTreeState` and `GoVersion`, it means that Helm is installed successfully.
```

File: docs/production-installation-using-helm.md [hunk 2]
```diff
@@ -1,6 +1,208 @@
...
+
+## Verify the installation
+
+To check the performance of Chaos Mesh, execute the following command:
+
+```sh
+kubectl get po -n chaos-testing
+```
+
+The expected outputs are as follows:
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:110]:
```suggestion
The expected output is as follows:
```

File: docs/production-installation-using-helm.md [hunk 3]
```diff
@@ -1,6 +1,208 @@
...
+
+```sh
+NAME                                        READY   STATUS    RESTARTS   AGE
+chaos-controller-manager-69fd5c46c8-xlqpc   1/1     Running   0          2d5h
+chaos-daemon-jb8xh                          1/1     Running   0          2d5h
+chaos-dashboard-98c4c5f97-tx5ds             1/1     Running   0          2d5h
+```
+
+If your actual outputs match the expected outputs, Chaos Mesh is installed successfully.
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:119]: 每个 name 信息和 Age 信息不可能完全一致的。"match" 是不准确的。参考：https://github.com/chaos-mesh/website/pull/81/files#diff-05a6a3b3e4820a5eb814a7c94bfb45d818c9b91fb64f399274ddaa543417fb01R27

File: docs/production-installation-using-helm.md [hunk 4]
```diff
@@ -1,6 +1,208 @@
...
+
+```bash
+helm uninstall chaos-mesh -n chaos-testing
+```
+
+## FAQs
+
+### How can I install the latest version of Chaos Mesh?
+
+The `helm/chaos-mesh/values.yaml` file defines the mirror of the latest version (the master branch).To install the latest version of Chaos Mesh, execute the following command:
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:192]:
```suggestion
The `helm/chaos-mesh/values.yaml` file defines the mirror of the latest version (the master branch). To install the latest version of Chaos Mesh, execute the following command:
```

File: docs/production-installation-using-helm.md [hunk 5]
```diff
@@ -1,6 +1,208 @@
...
+chaos-controller-manager-69fd5c46c8-xlqpc   1/1     Running   0          2d5h
+chaos-daemon-jb8xh                          1/1     Running   0          2d5h
+chaos-dashboard-98c4c5f97-tx5ds             1/1     Running   0          2d5h
+```
+
+If your actual outputs match the expected outputs, Chaos Mesh is installed successfully.
+
+:::note
+
+If the `STATUS` of your actual outputs is not `Running`, then execute the following command to check the Pod details, and troubleshoot issues according to the error information.
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:123]: outputs --> output

File: docs/production-installation-using-helm.md [hunk 6]
```diff
@@ -1,6 +1,181 @@
...
+:::
+
+## Install Chaos Mesh using Helm
+
+### Step 1: Add Chaos Mesh repository
+
+Add the Chaos Mesh repository to the Helm repository:
+
+```bash
+help repo add chaos-mesh https://charts.chaos-mesh.org
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:43]:
```suggestion
helm repo add chaos-mesh https://charts.chaos-mesh.org
```

File: docs/production-installation-using-helm.md [hunk 7]
```diff
@@ -1,6 +1,181 @@
...
+
+```bash
+# Default to /var/run/docker.sock
+helm install chaos-mesh chaos-mesh/chaos-mesh-n=chaos-testing
+```
+
+#### Containerd
+
+```bash
+help install chaos-mesh chaos-mesh-mesh-n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:88]:
```suggestion
helm install chaos-mesh chaos-mesh-mesh-n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
```

File: docs/production-installation-using-helm.md [hunk 8]
```diff
@@ -1,6 +1,181 @@
...
+
+```bash
+helm uninstall chaos-mesh -n chaos-testing
+```
+
+## FAQs
+
+### How can I install the latest version of Chaos Mesh?
+
+The `helm/chaos-mesh/values.yaml` file defines the image of the latest version (the master branch).To install the latest version of Chaos Mesh, execute the following command:
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:165]:
```suggestion
The `helm/chaos-mesh/values.yaml` file defines the image of the latest version (the master branch). To install the latest version of Chaos Mesh, execute the following command:
```

File: docs/common/verify-installation.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
+To check the performance of Chaos Mesh, execute the following command:
```
Comment (TomShawn) [inline docs/common/verify-installation.md:1]:
```suggestion
To check the running status of Chaos Mesh, execute the following command:
```

File: docs/common/verify-installation.md [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+The expected output is as follows:
+
+```sh
+NAME                                        READY   STATUS    RESTARTS   AGE
+chaos-controller-manager-69fd5c46c8-xlqpc   1/1     Running   0          2d5h
+chaos-daemon-jb8xh                          1/1     Running   0          2d5h
+chaos-dashboard-98c4c5f97-tx5ds             1/1     Running   0          2d5h
+```
+
+If your actual output is similar to the expected output with `Version`, `GitCommit`, `GitTreeState`, and `GoVersion`, it means that Helm is installed successfully.
```
Comment (TomShawn) [inline docs/common/verify-installation.md:16]:
```suggestion
If your actual output is similar to the expected output with `NAME`, `READY`, `STATUS`, `RESTARTS`, and `AGE`, it means that Helm is installed successfully.
```

File: docs/production-installation-using-helm.md [hunk 11]
```diff
@@ -1,6 +1,181 @@
...
+
+helm install chaos-mesh helm/chaos-mesh -n=chaos-teting
+```
+
+### How can I disable the safe mode?
+
+The safe mode is enabled by default. To disable the safe mode, specify `dashboard.securityMode` as `false` during the installation or upgrade:
+
+```bash
+help install chaos-mesh helm/chaos-mesh -n=chaos-testing --set dashboard.securityMode=false
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:180]:
```suggestion
helm install chaos-mesh helm/chaos-mesh -n=chaos-testing --set dashboard.securityMode=false
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: docs/production-installation-using-helm.md [hunk 12]
```diff
@@ -1,6 +1,181 @@
...
+
+### Step 4: Install Chaos Mesh in different environments
+
+Because socket paths are listened to by the daemons of different running containers, you need to set different values for socket paths during installation. You can execute the following installation commands according to different environments.
+
+#### Docker
+
+```bash
+# Default to /var/run/docker.sock
+helm install chaos-mesh chaos-mesh/chaos-mesh-n=chaos-testing
```
Comment (g1eny0ung) [inline docs/production-installation-using-helm.md:82]: There is a typo here and in the following commands. Should be `helm install chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing`.

File: docs/production-installation-using-helm.md [hunk 13]
```diff
@@ -1,6 +1,181 @@
...
+:::note
+
+If you have upgraded Chaos Mesh in a non-Docker environment, you need to add the corresponding parameters as described in [Step 4: Install Chaos Mesh in different environments](#step-4-install-chaos-mesh-in-different-environments).
+
+:::
+
+To modify the configuration, set different values according to your need. For example, execute the following command to upgrade and uninstall `chaos-dashboard`:
+
+```bash
+helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing --set dashboard.cree=false
```
Comment (g1eny0ung) [inline docs/production-installation-using-helm.md:134]: `dashboard.create=false`

File: docs/production-installation-using-helm.md [hunk 14]
```diff
@@ -1,6 +1,181 @@
...
+For more values and their usages, refer to [all values](https://github.com/chaos-mesh/chaos-mesh/blob/master/helm/chaos-mesh/values.yaml).
+
+:::
+
+:::caution
+
+Currently, the latest CustomResourceDefinition (CRD) is not applied during the Helm upgrading, which might cause errors. To avoid this situation, you can apply the latest CRD manually:
+
+<PickVersion className="language-bash">
+curl-sSL https://mirrors.chaos-mesh.org/latest/crd.yaml | kubectl apply -f -
```
Comment (g1eny0ung) [inline docs/production-installation-using-helm.md:148]: `curl -sSL`

File: docs/production-installation-using-helm.md [hunk 15]
```diff
@@ -1,6 +1,181 @@
...
+
+## FAQs
+
+### How can I install the latest version of Chaos Mesh?
+
+The `helm/chaos-mesh/values.yaml` file defines the image of the latest version (the master branch). To install the latest version of Chaos Mesh, execute the following command:
+
+```bash
+# Clone repository
+git clone https://github.com/chaos-mesh/chaos-messh.git
```
Comment (g1eny0ung) [inline docs/production-installation-using-helm.md:169]: An extra `s`.

`git clone https://github.com/chaos-mesh/chaos-mesh.git`

File: docs/production-installation-using-helm.md (hunk 12 above)
Comment (TomShawn) [reply docs/production-installation-using-helm.md:82]:
```suggestion
helm install chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing
```

File: docs/production-installation-using-helm.md (hunk 13 above)
Comment (TomShawn) [reply docs/production-installation-using-helm.md:134]:
```suggestion
helm upgrade chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing --set dashboard.create=false
```

File: docs/production-installation-using-helm.md (hunk 14 above)
Comment (TomShawn) [reply docs/production-installation-using-helm.md:148]:
```suggestion
curl -sSL https://mirrors.chaos-mesh.org/latest/crd.yaml | kubectl apply -f -
```

File: docs/production-installation-using-helm.md (hunk 15 above)
Comment (TomShawn) [reply docs/production-installation-using-helm.md:169]:
```suggestion
git clone https://github.com/chaos-mesh/chaos-mesh.git
```

**Comment by TomShawn** [comment]
@g1eny0ung PTAL again. thanks!

File: docs/production-installation-using-helm.md [hunk 16]
```diff
@@ -1,6 +1,181 @@
...
+#### Containerd
+
+```bash
+helm install chaos-mesh chaos-mesh-mesh-n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
+```
+
+#### K3s
+
+```bash
+helm install chaos-mesh chaos-mesh-n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/k3s/containerd/containerd.sock
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:94]:
```suggestion
helm install chaos-mesh chaos-mesh -n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/k3s/containerd/containerd.sock
```

File: docs/production-installation-using-helm.md [hunk 17]
```diff
@@ -1,6 +1,181 @@
...
+
+```bash
+# Default to /var/run/docker.sock
+helm install chaos-mesh chaos-mesh/chaos-mesh -n=chaos-testing
+```
+
+#### Containerd
+
+```bash
+helm install chaos-mesh chaos-mesh-mesh-n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
```
Comment (TomShawn) [inline docs/production-installation-using-helm.md:88]:
```suggestion
helm install chaos-mesh chaos-mesh-mesh -n=chaos-testing --set chaosDaemon.runtime=containerd --set chaosDaemon.socketPath=/run/containerd/containerd.sock
```

**Pull request merged**